src/components/DatePicker/Month/Month.tsx

```diff
@@ -42,7 +35,7 @@ const Month: React.FC<MonthProps> = ({ year, month, onDaySelect, selectedDays, d
   const weeksWithDays = React.useMemo<WeekRow[]>(() => {
     const monthDate = currentDay.month(month).year(year).date(1);
     const daysOfMonth = monthDate.daysInMonth();
-    const startDay = monthDate.day();
+    const startDay = monthDate.day() ? monthDate.day() : 7;
```

Comment:
what kind of sorcery is this? i don't clearly understand the logic here

Reply:
check latest commit for documentation